py/sentry_relay/processing.py

```diff
@@ -292,11 +322,14 @@ def normalize_global_config(config):
     normalized one.
 
     :param config: the global config to validate.
+    :param json_dumps: a function that stringifies python objects
+    :param json_loads: a function that parses and converts JSON strings
     """
-    serialized = json.dumps(config)
+    serialized = json_dumps(config)
     normalized = rustcall(lib.normalize_global_config, encode_str(serialized))
     rv = decode_str(normalized, free=True)
     try:
-        return json.loads(rv)
-    except json.JSONDecodeError:
+        return json_loads(rv)
+    except:  # noqa
```

Comment:
this should be `except Exception:` if the intent is to catch all user-space exceptions

as written this catches `KeyboardInterrupt`, `SystemExit`, `SystemError`, etc. which _should_ terminate the process